tests/report_test.sh

```diff
@@ -379,6 +379,11 @@ function test_save_data_to()
   output=$(save_data_to '/this/is/An/InvaLid/Path')
   ret="$?"
   assert_equals_helper "We expect an invalid path error" "$LINENO" "$ret" 1
-}
 
+  # Try to use an valid directory path
+  output=$(save_data_to './')
+  ret="$?"
+
+  assert_equals_helper "We expect an valid path" "$LINENO" "$ret" 0
```

Comment:
I want to see more checks here. Some cases:
1. Use an invalid path;
2. Try to write in a root folder;
3. Try to write in a path where the folder was not created.